build/distributions/CentOS/8/x86_64/bolt_pulp3_config.yaml

```
@@ -422,6 +420,10 @@ PowerTools:
     - name: elinks
     - name: lua-filesystem
     - name: lua-posix
+  modules:
+    - stream: javapackages-runtime
```

From the updates in `build/distributions/CentOS/8/x86_64/DVD/ks/dvd/include/simp_ks_base`, this looks like it should be:

```suggestion
    - stream: javapackages-runtime:201801
```